### ISSUE TITLE
add Molecular Nodes/MDAnalysis to hackathon

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/UGM2023#12]
@yuxuanzhuang started planning the Molecular Nodes/MDAnalysis Blender hackathon session in  [discord: #project-proposals: MDA/MN Collaboration](https://discord.com/channels/807348386012987462/1134027222999646339)

> I've compiled some of my thoughts on [MN hackathon session](https://miro.com/app/board/uXjVMmCOOaw=/?share_link_id=869549952816) (get password from @yuxuanzhuang or discord thread) :
> 
> We will be conducting the session from 10:00 to 15:00 WEST on September 29th. @BradyAJohnston  are welcome to chip in but I do understand if you can't participate for the entire duration.
> 
> Currently, I think there are several potential goals to consider:
> 
> *    Working on an optimal API
> *    Improving the code's performance.
> *    Creating a prototype to visualize analysis results.
> *    Better timeline management.
> *    Select atoms within Blender (which may require an advanced level of Blender knowledge).

Add necessary materials to the [hackathon/blender](https://github.com/MDAnalysis/UGM2023/tree/main/hackathon/blender) directory. You can also rename it from blender to MolecularNodes but then you need to change links in README.md file(s).

**Comment by orbeckst** [comment]
Quick note for working on this repository: If you are just touching files in the `blender` directory or do minor fixes in the READMEs then you don't need to wait for a review. Just make your changes. If you're modifying someone else's stuff, do a PR + wait for a review (just ping someone like  @fiona-naughton (who is in charge of hackathon things) @jennaswa (for overall organizational things)  @orbeckst  (if you just need someone quickly) or ask on discord).

**Comment by orbeckst** [comment]
In your outline I saw that you're also preparing a video for participants to watch in advance to get set up. I think that's a great idea. Can you raise a separate issue for that?

We would just put videos up on YouTube in the (very tiny) MDAnalysis channel.

**Comment by BradyAJohnston** [comment]
@orbeckst could I get write access to put some info into the blender README?

**Comment by orbeckst** [comment]
@BradyAJohnston I added you to the repo with write access.

**Comment by IAlibay** [comment]
@yuxuanzhuang we are planning to send out instructions to attendees on Monday - are there any further changes you wish to make ahead of this?

**Comment by yuxuanzhuang** [comment]
just waiting for the YouTube link and I think it's good to go!

**Comment by BradyAJohnston** [comment]
I've added some more notes on explaining data flow, and will add some more in the next day or two as well, will hopefully help people get started interfacing with Blender.

**Comment by IAlibay** [comment]
@BradyAJohnston - for the sake of ensuring we don't have things go out of synch across multiple editors, could you please commit via PR rather than directly to the main branch?

**Comment by orbeckst** [comment]
@IAlibay just fyi when we started work on the repo I gave the guideline above https://github.com/MDAnalysis/UGM2023/issues/12#issuecomment-1714766299 to make sure things were moving along quickly.

**Comment by IAlibay** [comment]
@orbeckst that's fair, sorry I didn't see that. However, given how close we are to the UGM, could we reconsider this policy?

**Comment by orbeckst** [comment]
Sure, just communicate to everyone who’s working on the repo.Am 9/24/23 um 09:08 schrieb Irfan Alibay ***@***.***>:﻿
@orbeckst that's fair, sorry I didn't see that. However, given how close we are to the UGM, could we reconsider this policy?

**Comment by IAlibay** [comment]
I've added branch protection so folks shouldn't be able to commit directly
unless they force it.

**Comment by BradyAJohnston** [comment]
Yep apologies @IAlibay was following earlier instructions, will ensure it goes via PR.
